pytorch_to_returnn/converter/converter.py

```diff
@@ -81,17 +82,37 @@ def __init__(self,
     """
     self._model_func = model_func
     self._inputs_np = inputs
-    inputs_data_kwargs = {} if inputs_data_kwargs is None else inputs_data_kwargs.copy()
-    if "feature_dim_axis" not in inputs_data_kwargs and not inputs_data_kwargs.get("sparse", False):
-      assert len(inputs.shape) >= 2  # (batch,feature|channel,...)
-      if inputs_data_kwargs.get("batch_dim_axis", 0) == 0:
-        inputs_data_kwargs["feature_dim_axis"] = 1  # Torch uses batch-feature-major by default
-    if "shape" not in inputs_data_kwargs:
-      # Assume feature is static, all other dynamic.
-      # Assume batch-major.
-      inputs_data_kwargs["shape"] = [
-        inputs.shape[i] if i == inputs_data_kwargs["feature_dim_axis"] else None
-        for i in range(1, len(inputs.shape))]
+    assert type(self._inputs_np) in (numpy.ndarray, dict, list), (
```

Comment:
This should not be restricted. You should just use the `nest` module for this (see other code which uses `nest`).